packages/related-content/index.tsx

```diff
@@ -35,9 +34,9 @@ const RelatedContent = ({
         ))}
       </div>
       <div className={s.cta} data-testid="wpl-cta-container">
-        <StandaloneLink appearance={appearance} href={cta.href}>
+        <a className={s.ctaText} href={cta.href}>
```

Comment:
Should we be using next/link here?

Reply:
Yes, yes. Fixed [here](https://github.com/hashicorp/react-components/pull/845/commits/54265c99800c4698dc83377a3d1dff09c48ae136).